calico_containers/pycalico/datastore.py

```diff
@@ -1058,7 +1009,7 @@ def get_endpoint(self, hostname=None, orchestrator_id=None,
             return eps.pop()
 
     @handle_errors
-    def set_endpoint(self, hostname, container_id, endpoint):
+    def set_endpoint(self, hostname, orchestrator_id, container_id, endpoint):
```

Comment:
Does set_endpoint actually need the various IDs ... I assume they are actually encapsulated in the endpoint object iteself and so all but the endpoint are superflous?